packages/terra-arrange/CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ Unreleased
 ------------------
 ### Changed
 * Converted component to use CSS modules
-
+* Updated nightwatch test scripts
```

Comment:
This is slated for release today. We can leave these here and update if we need to release tomorrow.